testdata/ansible/memcached-operator/test/kuttl/kuttl-test.yaml

```diff
@@ -0,0 +1,15 @@
+# This file is a bundle-ready kuttl TestSuite configuration file.
+# Full configuration documentation can be found here:
+# https://kudo.dev/docs/testing/reference.html#testsuite
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+testDirs:
+- v1alpha1
+# +kubebuilder:scaffold:kuttl:testDirs
```

Comment:
Could we add a comment over the marker as well?
Should users change or not that and etc?